GLMakie/src/GLAbstraction/AbstractGPUArray.jl

```diff
@@ -193,8 +193,10 @@ max_dim(t)       = error("max_dim not implemented for: $(typeof(t)). This happen
 
 function (::Type{T})(x::Observable; kw...) where T <: GPUArray
     gpu_mem = T(x[]; kw...)
-    on(x-> update!(gpu_mem, x), x)
-    gpu_mem
+    # TODO merge these and handle update tracking during contruction
```

Comment:
Why do we have this todo, and not just:
```julia
  on(data) do new_data
      update!(gpu_mem, new_data)
      gpu_mem.requires_update[] = true
  end
```
(I already have this locally, will push some changes like this later)

Reply:
I separated them so the update tracking could be removed again when using fps_renderloop